src/views/templates/details/tabs/scheduling/components/Descheduler.tsx

```diff
@@ -58,22 +63,33 @@ const Descheduler: React.FC<TemplateSchedulingGridProps> = ({ template }) => {
       </DescriptionListTermHelpText>
 
       <DescriptionListDescription>
-        {useDeschedulerInstalled && (
-          <Button
-            isInline
-            isDisabled={!isDeschedulerInstalled}
-            onClick={() =>
-              createModal(({ isOpen, onClose }) => (
-                <DeschedulerModal template={template} isOpen={isOpen} onClose={onClose} />
-              ))
-            }
-            variant="link"
-            iconPosition={'right'}
-          >
-            {isDeschedulerOn ? t('ON') : t('OFF')}
-            <PencilAltIcon className="co-icon-space-l pf-c-button-icon--plain" />
-          </Button>
-        )}
+        <Tooltip
+          content={
+            !isDeschedulerInstalled &&
+            t(
+              'To enable the descheduler, you must install the Kube Descheduler Operator from OperatorHub and enable one or more descheduler profiles.',
+            )
+          }
+          position="right"
+        >
+          <span>
+            <Button
+              isInline
+              isDisabled={!isDeschedulerInstalled}
```

Comment:
why, a user can't set descheduler and later install the operator ?

Reply:
What is not correct about the fact that the descheduler may not be installed? Of course it can be installed later, but it does not make sense to have this option enabled, if it is not installed.